f.ReadDir undefined (type *os.File has no field or method ReadDir, but does have Readdir)

**Describe the bug**
[A clear and concise description of what the bug is.]
It reports "net/net_linux.go:548:22: f.ReadDir undefined (type *os.File has no field or method ReadDir, but does have Readdir)" when I run make build_test
**To Reproduce**
```go
// paste example code reproducing the bug you are reporting
```
Run make build_test on top dir.
**Expected behavior**
[A clear and concise description of what you expected to happen.]
Every testcase passed.
**Environment (please complete the following information):**
 - [√ ] Linux: [centos6]

**Additional context**
[Cross-compiling? Paste the command you are using to cross-compile and the result of the corresponding `go env`]
go1.15.6

--- a/net/net_linux.go
+++ b/net/net_linux.go
@@ -549,7 +549,7 @@ func getProcInodes(root string, pid int32, max int) (map[string][]inodeMap, erro
 		return ret, err
 	}
 	defer f.Close()
-	dirEntries, err := f.ReadDir(max)
+	dirEntries, err := readDir(f, max)
 	if err != nil {
 		return ret, err
 	}

--- a/net/net_linux_111.go
+++ b/net/net_linux_111.go
@@ -1,0 +1,12 @@
+//go:build !go1.16
+// +build !go1.16
+
+package net
+
+import (
+	"os"
+)
+
+func readDir(f *os.File, max int) ([]os.FileInfo, error) {
+	return f.Readdir(max)
+}

--- a/net/net_linux_116.go
+++ b/net/net_linux_116.go
@@ -1,0 +1,12 @@
+//go:build go1.16
+// +build go1.16
+
+package net
+
+import (
+	"os"
+)
+
+func readDir(f *os.File, max int) ([]os.DirEntry, error) {
+	return f.ReadDir(max)
+}

